main.py

```diff
@@ -885,6 +885,9 @@ def map_select():
                 #make it so that when you click play, it goes to the game loop
                 if play.checkForInput(mouse):
                     pygame.display.set_caption("Game")
+                    mixer.music.load("game/assets/audio/background-game.wav")
```

Comment:
you need to load all images/ music with the resource path function resource_path()